src/SelectPanel/SelectPanel.tsx

```diff
@@ -60,7 +60,14 @@ const focusZoneSettings: Partial<FocusZoneHookSettings> = {
 export function SelectPanel({
   open,
   onOpenChange,
-  renderAnchor = props => <DropdownButton {...props} />,
+  renderAnchor = props => {
+    const {children, ...rest} = props
+    return (
+      <Button trailingAction={TriangleDownIcon} {...rest}>
```

Comment:
@joshblack I think we can add this usage to the v36 discussion as an alternative to the removed `DropdownButton` ?